### PR TITLE
metallb-dashboard: Add error and updates monitoring

### PR DESCRIPTION
**Pull request by surajssd**
* Also I forgot to install the dashboard in previous PR. Now that is taken care of in 1st commit.
* Dashboard looks like this: https://snapshot.raintank.io/dashboard/snapshot/VSd2fULu673q1fuNr4TF5t9MtgD7C90v

### CODE REVIEW

File: pkg/components/metallb/manifests.go [hunk 1]
```diff
@@ -440,9 +440,182 @@ data:
...
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "rate(metallb_k8s_client_update_errors_total[1m])",
```
Comment (invidian) [inline pkg/components/metallb/manifests.go]: Maybe we could modify the label for those metrics, to only include pod name?

Comment (surajssd) [reply pkg/components/metallb/manifests.go]: Do you mean like this: https://snapshot.raintank.io/dashboard/snapshot/ojNAWQiZmHARtXOTZgEBlzGtAbjF07go ?

Show only pod names in the legend?

Comment (invidian) [reply pkg/components/metallb/manifests.go]: Yeah, I think it's more readable then, but I don't know the conventions, so if you think this is worse, please ignore :) Thanks!

Comment (surajssd) [reply pkg/components/metallb/manifests.go]: I think this is readable. Than before because earlier there was lot of data.

**Pull request merged**